### PR TITLE
docs: add sponsor badge and blurb to readme

### PR DESCRIPTION
**Pull request by nbolton**
Blocked by: #7533, #7535  {no-changelog-lint}

@sithlord48 What do you think? [Preview](https://github.com/deskflow/deskflow/blob/5f050508688c0ffb6c26c19809fa8206cd01cf54/README.md)

### CODE REVIEW

**Review by sithlord48 — COMMENTED** [review]
Looks good might consider putting a bit about synergy with the other clients maybe just a single line

[Synergy](https://symless.com/synergy) - ,some one line description

**Comment by nbolton** [comment]
> Looks good might consider putting a bit about synergy with the other clients maybe just a single line

Good idea. I also think "Collaborative" is more accurate than "Similar" since we all work together.

File: README.md [hunk 1]
```diff
@@ -200,3 +205,17 @@ versions across them and gathering other information.
 ## License
 
 This project is licensed under [GPL-2.0](LICENSE) with an [OpenSSL exception](LICENSE_EXCEPTION).
+
+## Relationship with Synergy 
+
+<a href="https://symless.com/synergy">
```
Comment (sithlord48) [inline README.md:211]: Unneeded  you only need the  line  for your image to show up

File: README.md [hunk 2]
```diff
@@ -200,3 +205,17 @@ versions across them and gathering other information.
 ## License
 
 This project is licensed under [GPL-2.0](LICENSE) with an [OpenSSL exception](LICENSE_EXCEPTION).
+
+## Relationship with Synergy
```
Comment (sithlord48) [inline README.md:209]: Consider removing this section and just putting the badge below our others. In the future if we have additional sponsors we can also put them in here.

Comment (nbolton) [reply README.md:209]: Sounds good.

**Review by sithlord48 — APPROVED** [review]

**Pull request merged**